x/ibc/03-connection/keeper/verify.go

```diff
@@ -19,7 +19,7 @@ func (k Keeper) VerifyClientConsensusState(
 	consensusHeight uint64,
 	proof commitmentexported.Proof,
 	consensusState clientexported.ConsensusState,
-) error {
+) (err error) {
```

Comment:
```suggestion
) error {
```

Reply:
err needs to be declared since `clientState` is already declared. Alternative approach would be to do `var err error`